toolchain/check/import_cpp.cpp

```diff
@@ -373,6 +374,74 @@ static auto ClangLookup(Context& context, SemIR::NameScopeId scope_id,
   return lookup;
 }
 
+// The name of the Carbon function a C++ constructor is mapped to.
+static constexpr llvm::StringLiteral CarbonConstructorFunctionMame = "Make";
```

Comment:
My understanding from the past discussion with Chandler and Richard is they want to use the class name for C++ constructors, not `Make`. This is specifically so that a class with a `Make` member is not a name conflict; the type name cannot lead to a conflict in C++ because that's reserved for the constructor.

Reply:
Note, https://docs.google.com/document/d/1_bD_WWkWaikPCxpZnzp8PAf-BFWkcTH-GJpw1GZt17E/edit?tab=t.0 was never reviewed, and I don't think it reflects a final result. If you want a decision from leads here, I would encourage opening the leads question and fleshing out what you need.

Reply:
Created a leads question: #5896.
Changed to use the class name instead of `Make`.